### ISSUE TITLE
Column styling doesn't seem to be working

### ISSUE THREAD

**Issue by PaulCombal** [aVadim483/fast-excel-writer#85]
The following code produces an empty file:

```php
        $excel = Excel::create(['test']);
        $style = [
            Style::FILL_COLOR => '#FFFF00'
        ];
        $excel->getSheet()->setColStyle('C', $style);
        $excel->getSheet()->setColOptions('C', $style);
        $excel->save($fileName);
```

The expected result is color on the column C.

Is there something else to it?

Thank you

**Comment by aVadim483** [comment]
Yes, there is incorrect behavior, now the style is applied not to the column, but to the cells of the column, if there are any values ​​written there

I will fix it

**Comment by PaulCombal** [comment]
Do you have any idea when you could look it up?

**Comment by aVadim483** [comment]
Yesterday I released a new version 6.1, where the behavior of old functions has been changed and new functions have been added. Calls to old functions were not obvious.

Now, if you need to change the style of the entire column (or several columns), you should use ```setColStyle()``` or ```setColStyleArray()```, if you need to set the style only for the filled cells in the column, you should use ```setColDataStyle()``` or ```setColDataStyleArray()```. Similar functions exist for rows.

**Comment by PaulCombal** [comment]
I just saw that, I'm going to check it out right now! Once again, amazing work :)
